### ISSUE TITLE
Create "Contact Us" Page

### ISSUE THREAD

**Issue by johnr54321** [hackforla/311-data#447]
# Description
The purpose of this page is to put all user requests into tickets on our 311 Data - Support Project.  If this is too complicated, let me know and I can give you an email.

## Action Items

- [ ] Create Front End of Contact Us Page
- [x] Ensure that the issue goes to an email address or github 311 Data - Support Project

## Resources
**Figma Model**: https://www.figma.com/file/9RJJyaGAdT5hcRuv9PUMOY/311-Wireframe-v1.0?node-id=2288%3A96003
**Github Rest API**: https://developer.github.com/v3/

**Comment by johnr54321** [comment]
This issue relates to: https://github.com/hackforla/311-data/issues/378

**Comment by johnr54321** [comment]
I've added the github rest API so you can see if we can create issues from submissions rather than rely on someone's email address.

**Comment by rgao** [comment]
UPDATE: I implemented some backend code for creating issues using the github api. When I tried testing it by running the app, I ran into a few errors, notably an async error from using await.

```
TypeError: object NoneType can't be used in 'await' expression
[2020-03-28 08:09:21 -0700] [10385] [DEBUG] CORS: Request to '/githubrequest' matches CORS resource '/*'. Using options: {'origins': ['.*'], 'methods': 'DELETE, GET, HEAD, OPTIONS, PATCH, POST, PUT', 'allow_headers': ['.*'], 'expose_headers': None, 'supports_credentials': False, 'max_age': None, 'send_wildcard': False, 'automatic_options': True, 'vary_header': True, 'resources': '/*', 'intercept_exceptions': True, 'always_send': True}
```
I couldn't find a definitive answer. Anyone knows how to tackle this?

**Comment by rgao** [comment]
In the meantime I'm working on the design of the contact page according to Figma Model. I should be able to get that pushed tonight.

**Comment by adamkendis** [comment]
@rgao Can I see your backend code and the full error stacktrace? I got issue creation working with Postman, but ran into different errors doing it through a route on the sanic server. I'm using requests_async to make the POST request to the github api.

**Comment by adamkendis** [comment]
One more thing. It doesn't look like the github api makes it easy (or even possible) to create an issue and add it to a project board in one step. I think we'll also need to create a project card.

https://developer.github.com/v3/projects/cards/#create-a-project-card

The options I see are:

1. POST to github api to create new issue and get issue ID from github response. Then POST to github api to create a project card with `content_id=<new issue id>`, `content_type=Issue` 
2. POST to github api to create a new project card with the form data dropped into the card `note` field.

**Comment by rgao** [comment]
I'll send you a message on Slack for my back end code

**Comment by rgao** [comment]
OK, I finished... 

- react front-end design
- back-end for email integration

The email is sending from the back end, but I'm pretty sure I made a lot of errors on the front-end portion of that so it's not connecting to the back-end when the submit button is clicked.
```
axios.get('/emailservice', { email, message })
      .then(res => {
        console.log(res);
        console.log(res.data);
      });
```
This is what I have for sending data to the back end. It doesn't look right; anyone knows how I should fix it?
